demo/drive.md

```diff
@@ -36,7 +36,7 @@ This example shows you how to use Tiny Drive for your file and image management.
   }
 </style>
 
-<script src="https://devpreview.tiny.cloud/demo/tinymce.min.js"></script>
+<script src="https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/4/tinymce.min.js"></script>
```

Comment:
I didn't realise we usually put api keys in the docs o.o

Reply:
They are in the page source for each page with a codepen. However this page doesn't have a codepen so we're doing this.